domain/src/main/kotlin/no/nav/su/se/bakover/domain/visitor/LagBrevRequestVisitor.kt

```diff
@@ -528,20 +502,63 @@ class LagBrevRequestVisitor(
             )
         }
 
-    private fun vedtakIngenEndringIYtelse(vedtak: Vedtak.IngenEndringIYtelse): Either<KunneIkkeLageBrevRequest, LagBrevRequest> = hentPersonOgNavn(
-        fnr = vedtak.behandling.fnr,
-        saksbehandler = vedtak.saksbehandler,
-        attestant = vedtak.attestant,
-    ).map {
-        requestIngenEndring(
-            personOgNavn = it,
-            beregning = vedtak.beregning,
-            fritekst = when (val b = vedtak.behandling) {
-                is Revurdering -> b.fritekstTilBrev
-                else -> ""
+    private fun revurderingIngenEndring(revurdering: Revurdering, beregning: Beregning) =
+        hentPersonOgNavn(
+            fnr = revurdering.fnr,
+            saksbehandler = revurdering.saksbehandler,
+            attestant = FinnAttestantVisitor().let {
+                revurdering.accept(it)
+                it.attestant
             },
-            harEktefelle = vedtak.behandlingsinformasjon.harEktefelle(),
-            uføregrunnlag = vedtak.behandling.grunnlagsdata.uføregrunnlag
         )
-    }
+            .zip(hentGjeldendeUtbetaling(revurdering.sakId, LocalDate.now(clock)))
+            .map { (personOgNavn, gjeldendeUtbetaling) ->
+                requestIngenEndring(
+                    personOgNavn = personOgNavn,
+                    beregning = beregning,
+                    fritekst = revurdering.fritekstTilBrev,
+                    harEktefelle = revurdering.grunnlagsdata.bosituasjon.harEktefelle(),
+                    uføregrunnlag = revurdering.grunnlagsdata.uføregrunnlag,
+                    gjeldendeMånedsutbetaling = gjeldendeUtbetaling,
+                )
+            }
+
+    private fun vedtakIngenEndringIYtelse(vedtak: Vedtak.IngenEndringIYtelse): Either<KunneIkkeLageBrevRequest, LagBrevRequest.VedtakIngenEndring> =
+        hentPersonOgNavn(
+            fnr = vedtak.behandling.fnr,
+            saksbehandler = vedtak.saksbehandler,
+            attestant = vedtak.attestant,
+        )
+            .zip(hentGjeldendeUtbetaling(vedtak.behandling.sakId, vedtak.periode.fraOgMed)) // TODO IA: her burde vi kanskje bruke iverksettelsestidspunktet e.l.
```

Comment:
Det vil vel være vedtak.opprettet, så den er lett tilgjengelig!

Reply:
Ja, det var en god ide!

Reply:
Skulle vel tro at vi her var interessert i beløpet som har blitt/ville blitt utbetalt i perioden vi har revurdert? Tidspunktet vedtaket ble opprettet på har vel ikke nødvendigvis noe med perioden det er gyldig for å gjøre. Tror det _egentlige_ ordet du er ute etter er `virkningstidspunkt` og ikke `iverksettelsestidspunkt`- i så måte var det som sto her tidligere `vedtak.periode.fraOgMed` mer korrekt i mine øyne.

Reply:
Tja, begge er vel egentlig upresise, for hvis man ser på brukerens perspektiv så er det et øyeblikksbilde. Det vi ønsker å kommunisere er at det vedkommende fikk forespeilet på sin forrige beregning fortsatt gjelder. Avhengig av om revurderingen er langt eller kort framover eller bakover i tid vil kanskje `iverksettelsestidspunkt` være mest riktig i noen tilfeller, mens `vedtak.periode.fraOgMed` i andre. Uten at jeg har 100% klart for meg alle casene så har jeg en følelse av at `iverksettelsestidspunkt` blir oftest riktig.

Jeg tror det er veldig vanskelig å få dette 100% nøyaktig. En riktigere totalløsning kunne vært å enten klare å fiske opp den forrige beregninga og hente tallet derfra, eller at brevet endres sånn at vi ikke prøver å destillere 135 mulige verdier ned til en setning.

Reply:
Ser nå at kommentaren min var lite konklusiv hverken ene eller andre veien.

Jeg tenker at det underliggende problemet helst bør løses gjennom at vi endrer på brevet. Samtidig så er det som implementeres i denne PR-en mye mer korrekt enn det som ligger i prod i dag, så med mindre dere er veldig uenige i implementasjonen så tenker jeg vi bare kan ta inn dette nå.

Reply:
Ser poenget ditt, og heller vel også mot at det egentlige problemet er hvordan vi utformer brevene.